packages/office-ui-fabric-react/src/components/SpinButton/SpinButton.styles.ts

```diff
@@ -195,7 +195,8 @@ export const getStyles = memoizeFunction((
       outline: '2px dashed transparent',
       selectors: {
         [HighContrastSelector]: {
-          borderColor: 'Highlight'
+          borderColor: 'Highlight',
+          outline: 'none'

```

Comment:
why not keep using outline here?

Reply:
The outline was creating a lot of noise, and Ben wanted it gone. See this screenshot:
![image](https://user-images.githubusercontent.com/16619843/39538522-edb9db80-4df1-11e8-829c-4235df70d466.png)